lib/puppet/provider/cisco_interface/cisco.rb

```diff
@@ -201,6 +216,74 @@ def ipv4_addr_mask_set
     end
   end
 
+  def stp_mst_cost
```

Comment:
As discussed in scrum, try to use the autogen code for the getters.

Example: https://github.com/cisco/cisco-network-puppet-module/blob/5ff241703001e69875a16bf3347039b0ade15646/lib/puppet/provider/cisco_vrf_af/nxapi.rb#L48

**NOTE** You will likely still need to override the setters.